README.md

```diff
@@ -1,2 +1,64 @@
+
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
 # Leaderboard
-Leaderboard for games played
+
+> A simple Leaderboard Application Project. An application that can help manage game scores
+## Built With
+
+- Html
+- CSS
+- Javascript
+- Webpack
+- vscode
+- Github
+
+## Live Demo (if available)
+
+[Live Demo Link]()
+
+
+## Getting Started
+
+**Built with a Mobile Firts Approach to maximise performance on Mobile devices**
+
+
+To get a local copy up and running follow these simple example steps.
+
+### Application can run on any Browser.
+
+###  A simple to do List Application Project. An application that can help a group or an individual Manage Day to Day tasks in Linear tracking manner. though mainly demonstrating the Usage of Webpack and ES6 Modules
+
+### Application has been tested for error and browser functionality
+
+
+
+
+
+## Authors
+
+👤 **Paul Malongo**
+
+- GitHub: [@canon20paul](https://github.com/canon20paul/)
+- Twitter: [@twitterhandle](paul malongo)
+- LinkedIn: [paulmalongo(paulmalongo)
```

Comment:
- [ ] [OPTIONAL] Kindly make sure that your social links work and direct the user to your social profiles.

Reply:
sure i will thanks